ui/packages/consul-ui/ember-cli-build.js

```diff
@@ -46,6 +52,13 @@ module.exports = function(defaults) {
       }
     }
   }
+  if(['production'].includes(env)) {
+    // everything apart from production is 'debug', including test
+    // which means this and everything it affects is never tested
+    babel.plugins.push(
+      ['strip-function-call', {'strip': ['Ember.runInDebug']}]
```

Comment:
awesome 🎉

Reply:
Worth noting that as `runInDebug` is noop-ed in default ember usage, so even without this addition, anything that `runInDebug` affects is never tested either.

The only addition here is to strip it instead of turning it into a noop so we don't ship dead code that is never run.